azkaban-common/src/test/java/azkaban/executor/ExecutionFlowDaoTest.java

```diff
@@ -170,6 +170,9 @@ public void testFetchEmptyRecentlyFinishedFlows() throws Exception {
     final List<ExecutableFlow> flows = this.executionFlowDao
         .fetchRecentlyFinishedFlows(RECENTLY_FINISHED_LIFETIME);
     assertThat(flows.size()).isEqualTo(0);
+
+    //Restore the clock
+    DateTimeUtils.setCurrentMillisOffset(0);
```

Comment:
Oh, so this test leaks DateTimeUtils offset? It would be safer to place the reset in an `@After` method to not cause side effects even if the test case fails.

Reply:
fair point, I will update the PR